app/views/govuk_publishing_components/components/docs/image_card.yml

```diff
@@ -66,14 +66,14 @@ examples:
         text: "Press release"
       heading_text: "Government does things"
       description: "Following a thorough review of existing procedure, a government body has today announced that further work is necessary."
-  with_extra_links:
+  with_extra_list:
```

Comment:
`with_extra_details`?

Reply:
I requesting this re-review a bit too soon, this should be sorted.